<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnumValue</class>
 <widget class="QWidget" name="EnumValue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>85</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EnumDetails</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="valueLabel">
          <property name="text">
           <string>value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="valueEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="enumDeleteButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
